### PR TITLE
#61 home page fixes

### LINKED ISSUE #61: Home page fixes
## Homepage stuff
- Add padding top to "Used by 1k+ students globally"
- change hero picture
- accordion subtitle padding bottom
- WhyWCC points title padding bottom (pb-2)
- Remove "Learn More" button in OurCourses section
- Make 1,2,3 courses into links, probably cards (card component when?? 👀)
- mobile: OurCourses, padding top 32px
- Make students anonymous in WhatOurStudentsSay, remove names

### PR DESCRIPTION
**Pull request by JagrajAulakh**
Closes #61

### CODE REVIEW

File: src/components/HomePage/OurCoursesSection.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
 import { Link } from "react-router-dom";
 import tempImg from "../../assets/images/HomePage/tempImg.jpg";
 import PageSection from "../PageSection";
+import Card from "../Card";
 
 function OurCoursesSection() {
+	const courses = [
+		{
+			title: "PYTHON: BEGINNERS",
+			body: "This course will teach you everything about Syntax, Loops, Lists and functions and give questions to solidify the concepts",
```
Comment (hassanuahmad) [inline src/components/HomePage/OurCoursesSection.js]: i thoguht yall didnt mention anything to do with loops or anything. just overall y this course is for u. you can check figma file for the kinda copyright we wanted to go for

File: src/components/HomePage/WhatOurStudentsSay.js [hunk 2]
```diff
@@ -36,13 +36,13 @@ function WhatOurStudentsSay() {
...
 			course: "Python Beginner",
 			reviewText:
 				"The course was really for as the notes were really clear and everyone on the discord was there to help and I would get answers immediately. I look forward to creating projects with these set of skills.",
 		},
 		{
-			name: "Ahmad",
+			name: "Student 4",
 			course: "Python Beginner",
 			reviewText:
 				"The course was really for as the notes were really clear and everyone on the discord was there to help and I would get answers immediately. I look forward to creating projects with these set of skills.",
```
Comment (hassanuahmad) [inline src/components/HomePage/WhatOurStudentsSay.js]: prolly wanna get rid of this, its exact the same review as student 3. it was just there for testing purposes

File: src/components/HomePage/WhatOurStudentsSay.js [hunk 3]
```diff
@@ -61,7 +61,7 @@ function WhatOurStudentsSay() {
 							className="p-6 mx-2.5 border rounded-lg"
 						>
 							<h1 className="h3-text mb-0">{eachReview.name}</h1>
```
Comment (hassanuahmad) [inline src/components/HomePage/WhatOurStudentsSay.js:57]: also if we can set the max height of the div so its doesn't change height based on the amount of text

Comment (JagrajAulakh) [reply src/components/HomePage/WhatOurStudentsSay.js:57]: true, will get on that 😉

File: src/components/HomePage/OurCoursesSection.js (hunk 1 above)
Comment (JagrajAulakh) [reply src/components/HomePage/OurCoursesSection.js]: done

File: src/components/HomePage/WhatOurStudentsSay.js (hunk 2 above)
Comment (JagrajAulakh) [reply src/components/HomePage/WhatOurStudentsSay.js]: done

File: src/components/HomePage/WhatOurStudentsSay.js (hunk 3 above)
Comment (JagrajAulakh) [reply src/components/HomePage/WhatOurStudentsSay.js:57]: done

**Review by hassanuahmad — APPROVED** [review]

**Pull request merged**